test/unit/test_formula.py

```diff
@@ -42,7 +46,16 @@ def test_generate_formula():
         'license': {'spdx_id': 'MIT'},
     }
 
-    formula = Formula.generate_formula_data(USERNAME, REPO_NAME, repository, CHECKSUM, INSTALL, TAR_URL, TEST)
+    formula = Formula.generate_formula_data(
+        USERNAME,
+        REPO_NAME,
+        repository,
+        CHECKSUM,
+        INSTALL,
+        TAR_URL,
+        DEPENDS_ON,
+        TEST,
```

Comment:
Because we are adding more parameters, can we please update line 33 to say `Tests that we generate the formula content correctly when all parameters are passed.` This will help clarify that this test tests **everything**, following tests will test when certain parameters are missing and that they are handled gracefully.